ethereumj-core/src/main/java/org/ethereum/sharding/crypto/Sign.java

```diff
@@ -0,0 +1,48 @@
+package org.ethereum.sharding.crypto;
+
+import java.math.BigInteger;
+import java.util.Objects;
+
+/**
+ * Signature utilities
+ * Signature should be implemented using BLS
+ */
+public interface Sign {
```

Comment:
This interface lacks, at least, `recover` method that recovers a pub key from signature with usage of msg hash